### PR TITLE
Feature 469/directory split button closes #469

### LINKED ISSUE #469: FE> Split button on Directory page (S)
Definition of Done

- [x] combine all actions buttons for a team member into single split button

### PR DESCRIPTION
**Pull request by hwilliams314**

### CODE REVIEW

**Review by Bagurp — APPROVED** [review]

File: web-ui/src/components/split-button/SplitButton.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const SplitButton = ({toggleOnSelect=false, onClick, options}) => {
             <ArrowDropDownIcon />
           </Button>
         </ButtonGroup>
-        <Popper open={open} anchorEl={anchorRef.current} role={undefined} placement="bottom-end" transition disablePortal>
+        <Popper open={open} anchorEl={anchorRef.current} role={undefined} placement="bottom-end" transition >
```
Comment (mkimberlin) [inline web-ui/src/components/split-button/SplitButton.jsx:60]: What's the reason for this change?

Comment (hwilliams314) [reply web-ui/src/components/split-button/SplitButton.jsx:60]: it the disableProtal is in there the split button goes behind the picture of the person below and you can't see it

Comment (mkimberlin) [reply web-ui/src/components/split-button/SplitButton.jsx:60]: ok...does this impact where the popper opens?

Comment (hwilliams314) [reply web-ui/src/components/split-button/SplitButton.jsx:60]: nope, just the way it shows up

**Review by mkimberlin — APPROVED** [review]
